mppm/release-notes/release-notes.md

```diff
@@ -12,14 +12,14 @@ description: Release notes for Multiplayer Play Mode including new features, upd
 - Fixed the compilation error "'UnityPlayer' does not contain a definition for 'Role'" when the Dedicated Server package version doesn't match.
 
 ### Added
-- Avoid reconstructing virtual player windows through Play modes if their layouts in PlayMode and EditMode are the same. This helps retain anchoring in Windows.
+- The Virtual Player window now keeps the same layout when you switch between Play mode and Edit mode.
```

Comment:
It's specific to windows so I think it's worth mentioning it.. so more like "In Windows, the Virtual Player window now retains the anchoring when the same layouts are used when you switch between Play mode and Edit mode."

in other words the original description made a little more sense. it's the anchoring or window position that persists and not the layouts.

Reply:
Thanks! :)